docs/api/service-workers.md

```diff
@@ -66,8 +77,14 @@ Returns `Record<number, ServiceWorkerInfo>` - A [ServiceWorkerInfo](structures/s
 
 #### `serviceWorkers.getFromVersionID(versionId)`
 
-* `versionId` number
+* `versionId` number - ID of the service worker version
 
 Returns [`ServiceWorkerInfo`](structures/service-worker-info.md) - Information about this service worker
 
 If the service worker does not exist or is not running this method will throw an exception.
+
+#### `serviceWorkers.fromVersionID(versionId)` _Experimental_
```

Comment:
One suggestion from today's review, to make these method names slightly less confusing, make 2 new methods:

```
getInfoFromVersionID(versionId): ServiceWorkerInfo
getWorkerFromVersionID(versionId): ServiceWorkerMain
```

Then make `serviceWorkers.getFromVersionID` a deprecated alias for `getInfoFromVersionID`.